zyte_api/apikey.py

```diff
@@ -14,7 +14,7 @@ def get_apikey(key: Optional[str] = None) -> str:
     if key is not None:
         return key
     try:
-        return os.environ[ENV_VARIABLE]
+        return os.environ.get(ENV_VARIABLE)
```

Comment:
Why do you need to make this change?

Reply:
Sorry! I had not set the environment variable correctly and was getting an error. I have reverted the change.